packages/roosterjs-editor-core/lib/coreApi/triggerEvent.ts

```diff
@@ -5,8 +5,9 @@ import {
     PluginEventType,
     TriggerEvent,
 } from 'roosterjs-editor-types';
+import type { CompatiblePluginEventType } from 'roosterjs-editor-types/lib/compatibleTypes';
 
-const allowedEventsInShadowEdit = [
+const allowedEventsInShadowEdit: (PluginEventType | CompatiblePluginEventType)[] = [
```

Comment:
No, we do this on purpose. 

Compatible enums always have the same values with the const enum. So that at runtime, we just need to use one of them. We add the Compatible enums just for build time, to satisfy compiler.

And we need to avoid using Compatible enums as a value in roosterjs code and always do "import type" only otherwise it will pull in a lot of code to do bidirectional conversion and increase bundle size.